src/functionsDeployHelper.ts

```diff
@@ -140,14 +150,27 @@ export function logFilters(
   }
 }
 
-export function getFunctionsInfo(parsedTriggers: CloudFunctionTrigger[], projectId: string) {
-  const functionsInfo: CloudFunctionTrigger[] = [];
+export interface RegionMap {
+  [region: string]: CloudFunctionTrigger[];
+}
+
+/**
+ * Creates a map of regions to all the CloudFunctions being deployed
+ * to that region.
+ * @param projectId The project in use.
+ * @param parsedTriggers A list of all CloudFunctions in the deployment.
+ */
+export function createFunctionRegionMap(
+  projectId: string,
+  parsedTriggers: CloudFunctionTrigger[]
+): RegionMap {
+  const regionMap: RegionMap = {};
   _.forEach(parsedTriggers, (trigger) => {
```

Comment:
Please move to either `for...of` or `.forEach` from lodash. Don't have to do it everywhere, but I'd like us to do it in the places we're touching code.